package.json

```diff
@@ -3,25 +3,33 @@
   "version": "0.0.0",
   "private": true,
   "devDependencies": {
+    "coffeelint": "^1.16.0",
     "grunt": "^1.0.0",
+    "grunt-build-control": "^0.7.1",
+    "grunt-coffeelint": "0.0.16",
     "grunt-contrib-connect": "^1.0.2",
-    "grunt-contrib-watch": "^1.0.0",
     "grunt-contrib-copy": "^1.0.0",
     "grunt-contrib-jshint": "^1.1.0",
-    "load-grunt-tasks": "^3.5.2",
-    "grunt-build-control": "^0.7.1",
-    "grunt-coffeelint": "0.0.16",
-    "coffeelint": "^1.16.0"
+    "grunt-contrib-watch": "^1.0.0",
+    "load-grunt-tasks": "^3.5.2"
   },
   "engines": {
-    "node": ">=4"
+    "node": "^4",
+    "yarn": ">= 1.0.0"
   },
   "repository": {
     "type": "git",
     "url": "git@github.com:GabLeRoux/to-pull-or-not-to-pull.git"
   },
   "scripts": {
     "test": "grunt test",
-    "build": "grunt dist"
+    "build": "grunt dist",
+    "postsinstall": "node -e require('fs').symlinkSync(require('path').join(process.cwd(), 'node_modules/@bower_components'), require('path').join(process.cwd(), 'bower_components'), 'junction')"
```

Comment:
this is correct:

```
    "postinstall": "node -e \"try { require('fs').symlinkSync(require('path').join(process.cwd(), '../node_modules/@bower_components'), require('path').join(process.cwd(), 'public/components'), 'junction') } catch (e) { }\""
```

Reply:
Oh indeed, I failed at escaping the right quites, my bad 👍

Reply:
Hmm, but I'm not sure if wrapping everything in a `try` clause is the right thing to do, here's a failing example:

I have a `bower_components` folder that exists locally. It should be a `symlink`, but it's an actual folder. Muting the error is not the right thing here. We should continue if symlink already exist, but not if the symlink fails to create because a folder exist.

Anyway, that's kind of an edge case and I'm ok with it ;)